### PR TITLE
Using image streams with k8s resources

### PR DESCRIPTION
**Pull request by soltysh**
This is a docs for https://trello.com/c/BW0FS66Q/845-8-integrate-imagestreams-with-non-origin-resources. 

@ahardin-rh ptal, you can also read more about this in this article that should be published at some point in time: https://docs.google.com/document/d/1OI5uVoOcH59b-xB8J595OCabbdD3Oj98voWebLbHC8w/edit
This should be part of 3.6 release, as well.

### CODE REVIEW

**Comment by soltysh** [comment]
[(8) Integrate ImageStreams with non-origin resources](https://trello.com/c/BW0FS66Q/845-8-integrate-imagestreams-with-non-origin-resources)

**Comment by ahardin-rh** [comment]
@soltysh Thank you so much! I will make some minor style edits in a follow-up PR and get this mentioned in the OCP 3.6 release notes.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/image_policy.adoc#admin-guide-image-policy[Image Policy]
|Added details about using image streams in Kubernetes resources.
%
|xref:../dev_guide/managing_images.adoc#dev-guide-managing-images[Managing Images]
|Added a new section on xref:../dev_guide/managing_images.adoc#using-is-with-k8s[Using Image Streams with Kubernetes Resources], a Technology Preview feature.
%

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -350,6 +350,47 @@ metadata:
...
+[[using-is-with-k8s]]
+== Using Image Streams with Kubernetes resources (Technology Preview)
+Image Streams, being {product-title} native resources, work out of the box with
+all the rest of native resources available in {product-title}, such as
+xref:../dev_guide/builds/index.adoc#dev-guide-how-builds-work[Builds] or
+xref:../dev_guide/deployments/basic_deployment_operations.adoc#dev-guide-basic-deployment-operations[Deployments].
+Currently, it is also possible to make them work with native Kubernetes resources,
+such as xref:../../dev_guide/jobs.adoc#dev-guide-jobs[Jobs],
+xref:../architecture/core_concepts/deployments.adoc#replication-controllers[Replication Controllers],
+xref:../architecture/core_concepts/deployments.adoc#replication-controllers[Replication Controllers] or
```
Comment (stuartchuan) [inline dev_guide/managing_images.adoc]: Seems duplication with above line. Does you want to refer 'Replicaset' ?

Comment (stuartchuan) [inline dev_guide/managing_images.adoc]: Seems fixed by https://github.com/openshift/openshift-docs/pull/4927

Comment (soltysh) [inline dev_guide/managing_images.adoc]: Yes, I've meant Replica Sets here, I've opened https://github.com/openshift/openshift-docs/pull/4935 to add RS there.

Comment (soltysh) [inline dev_guide/managing_images.adoc]: Thanks for noticing this!

**Pull request merged**